### PR TITLE
Replace links

### PR DESCRIPTION
**Pull request by PadowYT2**

### CODE REVIEW

File: modules/Core/pages/panel/api.php [hunk 1]
```diff
@@ -233,7 +233,7 @@
             'API' => $language->get('admin', 'api'),
             'PAGE' => PANEL_PAGE,
             'API_INFO' => $language->get('admin', 'api_info', [
-                'pluginLinkStart' => '<a href="https://plugin.namelessmc.com" target="_blank">',
```
Comment (samerton) [inline modules/Core/pages/panel/api.php:236]: I think this should stay as `plugin.namelessmc.com`

File: README.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ The following list is a brief summary of the features available in v2 pre-releas
 - 🎮 Minecraft integration
   - Display your Java edition or Bedrock servers statuses
   - Account verification using MCAssoc or the NamelessMC plugin
-  - [In-game plugin](https://www.spigotmc.org/resources/nameless-plugin-for-v2.59032)
+  - [In-game plugin](https://www.spigotmc.org/resources/59032)
```
Comment (samerton) [inline README.md:20]: Can this change to `plugin.namelessmc.com`?

**Review by samerton — CHANGES_REQUESTED** [review]

File: modules/Core/pages/panel/api.php (hunk 1 above)
Comment (PadowYT2) [reply modules/Core/pages/panel/api.php:236]: But then, in README, should it be that?

File: README.md (hunk 2 above)
Comment (Derkades) [reply README.md:20]: The plugin will probably need to move away from spigot for the next update so this is a good idea

**Pull request merged**